test/support/test_complex.h

```diff
@@ -26,6 +26,24 @@
 #include <type_traits>
 #include <cassert>
 
+#if !_MSVC_STL_CPP17_COMPLEX_CONSTEXPR_FNC_BROKEN
+#    define STD_COMPLEX_TESTS_STATIC_ASSERT(...) static_assert(__VA_ARGS__)
+#else
+template <typename T>
+void
+__complex_tests_assert(T condition)
```

Comment:
Seems like you don't have the `static_assert` without message (I count empty message as a "message"). Probably you don't need this function at all.